data/templates/css/wikimedia.scss

```diff
@@ -76,253 +77,317 @@ div[style*="float:right"] {
+        	margin-top: $spacing-newh4;
+        }
+
+        h5 {
+        	font: 700 1.2rem $font-secondary;  // 24px // Root: 20px
+        	margin-top: $spacing-newh5;
+        }
+
+        h6 {
+        	font: 400 1.0rem $font-secondary;  // 20px // Root: 20px
+        	margin-top: $spacing-newh6;
+        	font-style: italic;
+        }
+
         sup {
             font-size: xx-small;
             vertical-align: super;
         }
-    }
-    div > table {
-        margin-bottom: $rhythm;
-    }
-    ul {
-        list-style: disc;
-        margin-left: 30px;
-        li {
-            margin-bottom: 4px;
+
+        p, ul, ol, .dablink {
+            margin-bottom: $spacing;
+            margin-top: $spacing;
         }
-    }
-    ol {
-        list-style: decimal;
-        margin-left: 30px;
-    }
-    .gallery {
-        li.gallerybox {
-            width: auto !important;
-            display: inline-block;
-            margin-left: 4px;
-            div, .gallerytext p, .gallerytextwrapper {
-                width: auto !important;
-            }
-            .thumb {
-                float: none;
-                clear: none;
-                img {
-                    width: 100%;
-                }
+
+        ul {
+            list-style: disc;
+            margin-left: 30px;
+            ul, li {
+                margin-bottom: 5px;
+                width: 100%;
             }
         }
-        li.gallerycaption {
-            text-align: center;
-            display: block;
-            font-weight: bold;
+        ol {
+            list-style: decimal;
+            margin-left: 30px;
         }
-    }
-    .plainlist {
-        ul, ol {
-            list-style: none;
-            margin-bottom: 0;
-            margin-left: 0;
+
+        table {
+        	color: $text-color-body;
         }
-    }
-    .rellink {
-        font-style: italic;
-        margin-bottom: $rhythm;
-        margin-top: 0;
-    }
-    .navbox, div > table[style] {
-        background: #fff;
-        border: 5px solid #fff !important;
-        border-collapse: collapse;
-        clear: both;
-        font: inherit;
-        font-size: 88% !important;
-        margin-bottom: $rhythm;
-        text-align: center;
-        width: 100%;
-        @include box-shadow(0 0 6px 0 rgba(#000, 0.3));
-        @include box-sizing(border-box);
-        color: $gray;
-        td, th {
-            border: 2px solid #fff;
+
+        .gallery {
+        	margin: 0 0 0 0 !important;
+
+        	img {
+        		width: 100%;
+        	}
+
+        	div, p {
+        		margin: 0 0 0 0 !important;
+        	}
+
+            .gallerybox {
+                width: 100% !important;
+                display: inline-block;
+                margin-bottom: 20px;
+
+                div, .gallerytext p, .gallerytextwrapper {
+                    width: auto !important;
+                }
+
+                .gallerytext {
+                	font-family: $font-secondary; // 16 px
```

Comment:
idem! No tabs!

Note that you can make a commit that fixes this, and then use the same interactive rebase trick to unify these fixes in this same commit.